.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,73 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
```

Comment:
Not a big deal, but for future reference.... I have been leaving this header out of the `scorecard.yml` and instead just using the line below. That's why you won't see it in some of the repositories. But it isn't a big deal either way, because GitHub made it available for free with no license restrictions (as far as I can tell).